1-js/02-first-steps/06-type-conversions/article.md

```diff
@@ -1,74 +1,72 @@
-# Type Conversions
+# Conversiones de Tipos
 
-Most of the time, operators and functions automatically convert the values given to them to the right type. This is called "type conversion".
+La mayoría de las veces, los operadores y funciones convierten automáticamente los valores que se les pasan al tipo correcto. Esto es llamado "conversión de tipo".
 
-For example, `alert` automatically converts any value to a string to show it. Mathematical operations convert values to numbers.
+Por ejemplo, `alert` convierte automáticamente cualquier valor a string para mostrarlo. Las operaciones matemáticas convierten los valores a números.
 
-There are also cases when we need to explicitly convert a value to the expected type.
+Tmabién hay casos donde necesitamos convertir de manera explícita un valor al tipo esperado.
```

Comment:
`También`

Reply:
It seems you reviewed the wrong commit. That was fixed in the latest one. 

I will update this PR with the translation of task and solution.

Reply:
@tikoflano It keeps showing "Tmabién" and other comments. Can you confirm that you updated the parts I commented you about?